doc/rosenpass.1

```diff
@@ -0,0 +1,106 @@
+.Dd $Mdocdate$
+.Dt ROSENPASS 1
+.Os
+.Sh NAME
+.Nm rosenpass
+.Nd builds post-quantum-secure VPNs with WireGuard
```

Comment:
Can we mention that it performs a key exchange specifically and that it can work without WG?

Reply:
How would you phrase it? I can only come up with formulations, that would make rosenpass sound like it's own VPN protocol.

Reply:
> enhances WireGuard with post-quantum secure key-exchange

?